lib/cocina/models/request_file.rb

```diff
@@ -7,6 +7,7 @@ module Models
     # See http://sul-dlss.github.io/cocina-models/maps/File.json
     class RequestFile < Struct
       include FileAttributes
+      attribute :externalIdentifier, Types::Strict::String.meta(omittable: true)
```

Comment:
Can you drop a comment here about when and how this is used, because it might be confusing.